TrackingAntennaV3/main/Magnetometer.hpp

```diff
@@ -1,20 +1,23 @@
-/**
- * @file   Magnetometer.hpp
  * @Author Michael Lenover (WARG)
  * @date   January 5, 2017
  * @brief  Magnetometer initialization functions
+ *     Connection based off of the Arduino ethernet library.
+ *     Uses DHCP so no need to know the IP of the shield once connected.
  *         Connection based off of the Arduino ethernet library.
  *         Uses DHCP so no need to know the IP of the shield once connected.
  * @see https://www.arduino.cc/en/Reference/Ethernet
  */
 
-#ifndef MAGNETOMETER
-#define MAGNETOMETER
-
 #define COMPASS_ID 12345
```

Comment:
What is this compass id? Is this meant for I2C? then declare it as COMPASS_I2C_ID to remove any confusion